[OnlineJudge][INFO] [2016-06-21 13:56:29] org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(317) | Loading XML bean definitions from class path resource [spring.xml]
[OnlineJudge][INFO] [2016-06-21 13:56:29] org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(317) | Loading XML bean definitions from class path resource [spring-hibernate.xml]
[OnlineJudge][INFO] [2016-06-21 13:56:30] org.springframework.context.support.AbstractApplicationContext.prepareRefresh(578) | Refreshing org.springframework.context.support.GenericApplicationContext@4a41e2a3: startup date [Tue Jun 21 13:56:30 CST 2016]; root of context hierarchy
[OnlineJudge][INFO] [2016-06-21 13:56:30] org.springframework.core.io.support.PropertiesLoaderSupport.loadProperties(172) | Loading properties file from class path resource [config.properties]
[OnlineJudge][INFO] [2016-06-21 13:56:30] org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.<init>(153) | JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[OnlineJudge][INFO] [2016-06-21 13:56:31] com.mchange.v2.log.MLog$1.run(124) | MLog clients using log4j logging.
[OnlineJudge][INFO] [2016-06-21 13:56:32] com.mchange.v2.c3p0.C3P0Registry.banner(248) | Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[OnlineJudge][INFO] [2016-06-21 13:56:33] org.hibernate.annotations.common.Version.<clinit>(37) | HCANN000001: Hibernate Commons Annotations {4.0.1.Final}
[OnlineJudge][INFO] [2016-06-21 13:56:33] org.hibernate.Version.logVersion(41) | HHH000412: Hibernate Core {4.1.4.Final}
[OnlineJudge][INFO] [2016-06-21 13:56:33] org.hibernate.cfg.Environment.<clinit>(239) | HHH000206: hibernate.properties not found
[OnlineJudge][INFO] [2016-06-21 13:56:33] org.hibernate.cfg.Environment.buildBytecodeProvider(342) | HHH000021: Bytecode provider name : javassist
[OnlineJudge][INFO] [2016-06-21 13:56:34] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(522) | Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 5, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 30000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hgekx79hvi19piihshdz|1745d779, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hgekx79hvi19piihshdz|1745d779, idleConnectionTestPeriod -> 30, initialPoolSize -> 5, jdbcUrl -> jdbc:mysql://localhost:3306/djangodb?useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 30, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 5, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[OnlineJudge][INFO] [2016-06-21 13:56:35] org.hibernate.dialect.Dialect.<init>(122) | HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
[OnlineJudge][INFO] [2016-06-21 13:56:35] org.hibernate.engine.transaction.internal.TransactionFactoryInitiator.initiateService(68) | HHH000399: Using default transaction strategy (direct JDBC transactions)
[OnlineJudge][INFO] [2016-06-21 13:56:35] org.hibernate.hql.internal.ast.ASTQueryTranslatorFactory.<init>(48) | HHH000397: Using ASTQueryTranslatorFactory
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(182) | HHH000228: Running hbm2ddl schema update
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(193) | HHH000102: Fetching database metadata
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(205) | HHH000396: Updating schema
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(65) | HHH000261: Table found: djangodb.auth_group_permissions
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(66) | HHH000037: Columns: [id, permission_id, group_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(68) | HHH000108: Foreign keys: [auth_group_permissions_group_id_b120cbf9_fk_auth_group_id, auth_group_permissi_permission_id_84c5c92e_fk_auth_permission_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(69) | HHH000126: Indexes: [primary, auth_group_permissions_group_id_0cd325b0_uniq, auth_group_permissi_permission_id_84c5c92e_fk_auth_permission_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(65) | HHH000261: Table found: djangodb.auth_user_groups
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(66) | HHH000037: Columns: [id, group_id, user_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(68) | HHH000108: Foreign keys: [auth_user_groups_user_id_6a12ed8b_fk_auth_user_id, auth_user_groups_group_id_97559544_fk_auth_group_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(69) | HHH000126: Indexes: [auth_user_groups_user_id_94350c0c_uniq, primary, auth_user_groups_group_id_97559544_fk_auth_group_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(65) | HHH000261: Table found: djangodb.auth_user_user_permissions
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(66) | HHH000037: Columns: [id, permission_id, user_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(68) | HHH000108: Foreign keys: [auth_user_user_perm_permission_id_1fbb5f2c_fk_auth_permission_id, auth_user_user_permissions_user_id_a95ead1b_fk_auth_user_id]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(69) | HHH000126: Indexes: [auth_user_user_perm_permission_id_1fbb5f2c_fk_auth_permission_id, primary, auth_user_user_permissions_user_id_14a6b632_uniq]
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_group
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_permission
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_user
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_admin_log
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_content_type
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_group
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_permission
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_user
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_admin_log
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_content_type
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.auth_group (id integer not null auto_increment, name varchar(80) not null unique, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.auth_permission (id integer not null auto_increment, content_type_id integer not null, name varchar(255) not null, codename varchar(100) not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.auth_user (id integer not null auto_increment, password varchar(128) not null, last_login datetime, is_superuser boolean not null, username varchar(30) not null unique, first_name varchar(30) not null, last_name varchar(30) not null, email varchar(254) not null, is_staff boolean not null, is_active boolean not null, date_joined datetime not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.django_admin_log (id integer not null auto_increment, user_id integer not null, content_type_id integer, action_time datetime not null, object_id varchar(255), object_repr varchar(200) not null, action_flag smallint not null, change_message varchar(255) not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.django_content_type (id integer not null auto_increment, app_label varchar(100) not null, model varchar(100) not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: alter table onlinejudgedb.auth_permission add index FK9C0EA4C661939D68 (content_type_id), add constraint FK9C0EA4C661939D68 foreign key (content_type_id) references onlinejudgedb.django_content_type (id)
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Table 'onlinejudgedb.auth_permission' doesn't exist
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: alter table onlinejudgedb.django_admin_log add index FKB361767061939D68 (content_type_id), add constraint FKB361767061939D68 foreign key (content_type_id) references onlinejudgedb.django_content_type (id)
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Table 'onlinejudgedb.django_admin_log' doesn't exist
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: alter table onlinejudgedb.django_admin_log add index FKB3617670C44148EC (user_id), add constraint FKB3617670C44148EC foreign key (user_id) references onlinejudgedb.auth_user (id)
[OnlineJudge][ERROR] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Table 'onlinejudgedb.django_admin_log' doesn't exist
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(240) | HHH000232: Schema update complete
[OnlineJudge][INFO] [2016-06-21 13:56:36] org.hibernate.cache.spi.UpdateTimestampsCache.<init>(62) | HHH000250: Starting update timestamps cache at region: org.hibernate.cache.spi.UpdateTimestampsCache
[OnlineJudge][WARN] [2016-06-21 13:56:36] org.hibernate.cache.ehcache.AbstractEhcacheRegionFactory.getCache(180) | HHH020003: Could not find a specific ehcache configuration for cache named [org.hibernate.cache.spi.UpdateTimestampsCache]; using defaults.
[OnlineJudge][INFO] [2016-06-21 13:56:37] org.hibernate.cache.internal.StandardQueryCache.<init>(83) | HHH000248: Starting query cache at region: org.hibernate.cache.internal.StandardQueryCache
[OnlineJudge][WARN] [2016-06-21 13:56:37] org.hibernate.cache.ehcache.AbstractEhcacheRegionFactory.getCache(180) | HHH020003: Could not find a specific ehcache configuration for cache named [org.hibernate.cache.internal.StandardQueryCache]; using defaults.
[OnlineJudge][INFO] [2016-06-21 13:56:37] org.springframework.orm.hibernate4.HibernateTransactionManager.afterPropertiesSet(360) | Using DataSource [com.mchange.v2.c3p0.ComboPooledDataSource[ identityToken -> 1hgekx79hvi19piihshdz|1745d779, dataSourceName -> 1hgekx79hvi19piihshdz|1745d779 ]] of Hibernate SessionFactory for HibernateTransactionManager
[OnlineJudge][WARN] [2016-06-21 13:56:38] org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(143) | SQL Error: 1146, SQLState: 42S02
[OnlineJudge][ERROR] [2016-06-21 13:56:38] org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(144) | Table 'onlinejudgedb.auth_user' doesn't exist
[OnlineJudge][INFO] [2016-06-21 13:56:38] org.springframework.context.support.AbstractApplicationContext.doClose(960) | Closing org.springframework.context.support.GenericApplicationContext@4a41e2a3: startup date [Tue Jun 21 13:56:30 CST 2016]; root of context hierarchy
[OnlineJudge][INFO] [2016-06-21 13:57:36] org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(317) | Loading XML bean definitions from class path resource [spring.xml]
[OnlineJudge][INFO] [2016-06-21 13:57:37] org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(317) | Loading XML bean definitions from class path resource [spring-hibernate.xml]
[OnlineJudge][INFO] [2016-06-21 13:57:37] org.springframework.context.support.AbstractApplicationContext.prepareRefresh(578) | Refreshing org.springframework.context.support.GenericApplicationContext@365ce20: startup date [Tue Jun 21 13:57:37 CST 2016]; root of context hierarchy
[OnlineJudge][INFO] [2016-06-21 13:57:37] org.springframework.core.io.support.PropertiesLoaderSupport.loadProperties(172) | Loading properties file from class path resource [config.properties]
[OnlineJudge][INFO] [2016-06-21 13:57:37] org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.<init>(153) | JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[OnlineJudge][INFO] [2016-06-21 13:57:38] com.mchange.v2.log.MLog$1.run(124) | MLog clients using log4j logging.
[OnlineJudge][INFO] [2016-06-21 13:57:38] com.mchange.v2.c3p0.C3P0Registry.banner(248) | Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[OnlineJudge][INFO] [2016-06-21 13:57:39] org.hibernate.annotations.common.Version.<clinit>(37) | HCANN000001: Hibernate Commons Annotations {4.0.1.Final}
[OnlineJudge][INFO] [2016-06-21 13:57:39] org.hibernate.Version.logVersion(41) | HHH000412: Hibernate Core {4.1.4.Final}
[OnlineJudge][INFO] [2016-06-21 13:57:39] org.hibernate.cfg.Environment.<clinit>(239) | HHH000206: hibernate.properties not found
[OnlineJudge][INFO] [2016-06-21 13:57:39] org.hibernate.cfg.Environment.buildBytecodeProvider(342) | HHH000021: Bytecode provider name : javassist
[OnlineJudge][INFO] [2016-06-21 13:57:39] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(522) | Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 5, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 30000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hgekx79hvi2p4h1qauyvp|3ff5022b, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hgekx79hvi2p4h1qauyvp|3ff5022b, idleConnectionTestPeriod -> 30, initialPoolSize -> 5, jdbcUrl -> jdbc:mysql://localhost:3306/djangodb?useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 30, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 5, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.dialect.Dialect.<init>(122) | HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.engine.transaction.internal.TransactionFactoryInitiator.initiateService(68) | HHH000399: Using default transaction strategy (direct JDBC transactions)
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.hql.internal.ast.ASTQueryTranslatorFactory.<init>(48) | HHH000397: Using ASTQueryTranslatorFactory
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(182) | HHH000228: Running hbm2ddl schema update
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(193) | HHH000102: Fetching database metadata
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(205) | HHH000396: Updating schema
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(65) | HHH000261: Table found: djangodb.auth_group_permissions
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(66) | HHH000037: Columns: [id, permission_id, group_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(68) | HHH000108: Foreign keys: [auth_group_permissions_group_id_b120cbf9_fk_auth_group_id, auth_group_permissi_permission_id_84c5c92e_fk_auth_permission_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(69) | HHH000126: Indexes: [primary, auth_group_permissions_group_id_0cd325b0_uniq, auth_group_permissi_permission_id_84c5c92e_fk_auth_permission_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(65) | HHH000261: Table found: djangodb.auth_user_groups
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(66) | HHH000037: Columns: [id, group_id, user_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(68) | HHH000108: Foreign keys: [auth_user_groups_user_id_6a12ed8b_fk_auth_user_id, auth_user_groups_group_id_97559544_fk_auth_group_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(69) | HHH000126: Indexes: [auth_user_groups_user_id_94350c0c_uniq, primary, auth_user_groups_group_id_97559544_fk_auth_group_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(65) | HHH000261: Table found: djangodb.auth_user_user_permissions
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(66) | HHH000037: Columns: [id, permission_id, user_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(68) | HHH000108: Foreign keys: [auth_user_user_perm_permission_id_1fbb5f2c_fk_auth_permission_id, auth_user_user_permissions_user_id_a95ead1b_fk_auth_user_id]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.TableMetadata.<init>(69) | HHH000126: Indexes: [auth_user_user_perm_permission_id_1fbb5f2c_fk_auth_permission_id, primary, auth_user_user_permissions_user_id_14a6b632_uniq]
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_group
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_permission
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_user
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_admin_log
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_content_type
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_group
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_permission
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: auth_user
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_admin_log
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.DatabaseMetadata.getTableMetadata(120) | HHH000262: Table not found: django_content_type
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.auth_group (id integer not null auto_increment, name varchar(80) not null unique, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.auth_permission (id integer not null auto_increment, content_type_id integer not null, name varchar(255) not null, codename varchar(100) not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.auth_user (id integer not null auto_increment, password varchar(128) not null, last_login datetime, is_superuser boolean not null, username varchar(30) not null unique, first_name varchar(30) not null, last_name varchar(30) not null, email varchar(254) not null, is_staff boolean not null, is_active boolean not null, date_joined datetime not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.django_admin_log (id integer not null auto_increment, user_id integer not null, content_type_id integer, action_time datetime not null, object_id varchar(255), object_repr varchar(200) not null, action_flag smallint not null, change_message varchar(255) not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: create table onlinejudgedb.django_content_type (id integer not null auto_increment, app_label varchar(100) not null, model varchar(100) not null, primary key (id)) ENGINE=InnoDB
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Unknown database 'onlinejudgedb'
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: alter table onlinejudgedb.auth_permission add index FK9C0EA4C661939D68 (content_type_id), add constraint FK9C0EA4C661939D68 foreign key (content_type_id) references onlinejudgedb.django_content_type (id)
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Table 'onlinejudgedb.auth_permission' doesn't exist
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: alter table onlinejudgedb.django_admin_log add index FKB361767061939D68 (content_type_id), add constraint FKB361767061939D68 foreign key (content_type_id) references onlinejudgedb.django_content_type (id)
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Table 'onlinejudgedb.django_admin_log' doesn't exist
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(235) | HHH000388: Unsuccessful: alter table onlinejudgedb.django_admin_log add index FKB3617670C44148EC (user_id), add constraint FKB3617670C44148EC foreign key (user_id) references onlinejudgedb.auth_user (id)
[OnlineJudge][ERROR] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(236) | Table 'onlinejudgedb.django_admin_log' doesn't exist
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(240) | HHH000232: Schema update complete
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.cache.spi.UpdateTimestampsCache.<init>(62) | HHH000250: Starting update timestamps cache at region: org.hibernate.cache.spi.UpdateTimestampsCache
[OnlineJudge][WARN] [2016-06-21 13:57:40] org.hibernate.cache.ehcache.AbstractEhcacheRegionFactory.getCache(180) | HHH020003: Could not find a specific ehcache configuration for cache named [org.hibernate.cache.spi.UpdateTimestampsCache]; using defaults.
[OnlineJudge][INFO] [2016-06-21 13:57:40] org.hibernate.cache.internal.StandardQueryCache.<init>(83) | HHH000248: Starting query cache at region: org.hibernate.cache.internal.StandardQueryCache
[OnlineJudge][WARN] [2016-06-21 13:57:40] org.hibernate.cache.ehcache.AbstractEhcacheRegionFactory.getCache(180) | HHH020003: Could not find a specific ehcache configuration for cache named [org.hibernate.cache.internal.StandardQueryCache]; using defaults.
[OnlineJudge][INFO] [2016-06-21 13:57:41] org.springframework.orm.hibernate4.HibernateTransactionManager.afterPropertiesSet(360) | Using DataSource [com.mchange.v2.c3p0.ComboPooledDataSource[ identityToken -> 1hgekx79hvi2p4h1qauyvp|3ff5022b, dataSourceName -> 1hgekx79hvi2p4h1qauyvp|3ff5022b ]] of Hibernate SessionFactory for HibernateTransactionManager
[OnlineJudge][WARN] [2016-06-21 13:57:41] org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(143) | SQL Error: 1146, SQLState: 42S02
[OnlineJudge][ERROR] [2016-06-21 13:57:41] org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(144) | Table 'onlinejudgedb.auth_user' doesn't exist
[OnlineJudge][INFO] [2016-06-21 13:57:41] org.springframework.context.support.AbstractApplicationContext.doClose(960) | Closing org.springframework.context.support.GenericApplicationContext@365ce20: startup date [Tue Jun 21 13:57:37 CST 2016]; root of context hierarchy
